src/eeg_series.jl

```diff
@@ -157,140 +142,173 @@ function eeg_topoplot_series!(
         (
             colorrange = (q_min, q_max),
             interp_resolution = (128, 128),
-            contours = (levels = range(q_min, q_max; length = 7),),
+            contours = (levels = range(q_min, q_max; length = 7)),
```

Comment:
This is a potential bug IMHO, with the comma we get a named tuple, without, not.